### DISCUSSION TITLE
Genie with websockets in a GITPOD environment

### DISCUSSION THREAD

**Discussion by michaelfliegner** [GenieFramework/Genie.jl#567] [Show and tell]
**Is there a place in the docs for the following?**

This does the trick for Genie with websockets in a GITPOD environment:
In the app

`
        Genie.config.websockets_port = 8001
        Genie.config.websockets_exposed_port = 443
        Genie.config.websockets_exposed_host = "8001-$(replace(ENV["GITPOD_WORKSPACE_URL"],"https://"=> ""))"
`

In gitpod.yml

`
ports:
  - name: Web App
    description: The main application web server
    port: 8000
    visibility: public
    onOpen: open-browser

  - name: Web Socket
    description: Web Socket server UI synching
    port: 8001
    visibility: public
    onOpen: ignore

`

**Comment by essenciary** [comment]
Definitely - wanna add a new doc in the deployment series? https://github.com/GenieFramework/Genie.jl/tree/master/docs/src/tutorials

**Reply by michaelfliegner** [reply]
will do
